code/game/machinery/autolathe.dm

```diff
@@ -201,9 +201,6 @@
 		var/obj/item/stack/N = new being_built.build_path(A, multiplier)
 		N.update_icon()
 		N.autolathe_crafted(src)
-		for(var/obj/item/stack/S in (A.contents - N))
-			if(istype(S, N.merge_type))
-				N.merge(S)

```

Comment:
This is already done in /obj/item/stack/Initialize()
https://github.com/tgstation/tgstation/blob/master/code/game/objects/items/stacks/stack.dm#L43
It is also done slightly differently so we should probably go with the one defined in stack code.